usr/bin/whonix-workstation-firewall

```diff
@@ -309,7 +334,9 @@ nft_output() {
         if [ -n "$SYSTEMCHECK_USER" ]; then
           $nftables_cmd add rule inet filter output skuid "$SYSTEMCHECK_USER" inet daddr 127.0.0.1 counter accept
```

Comment:
Should there be a `$nftables_cmd add rule inet filter output skuid "$SYSTEMCHECK_USER" inet daddr ::1 counter accept` here?